### PR TITLE
Update gpu.md: Add RTX 3050 Ti and RTX 3050 Ti

### PR DESCRIPTION
**Pull request by bean5**
Seems strange that the laptop versions of 3050 and 3050 Ti would be supported but not the non-notebook, but this is what the page (https://developer.nvidia.com/cuda-gpus) says.

### CODE REVIEW

**Comment by jmorganca** [comment]
@bean5 thanks for the PR. Were you able to test these GPUs? Thanks!

File: docs/gpu.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Check your compute compatibility to see if your card is supported:
 | 9.0                | NVIDIA              | `H100`                                                                                                      |
 | 8.9                | GeForce RTX 40xx    | `RTX 4090` `RTX 4080 SUPER` `RTX 4080` `RTX 4070 Ti SUPER` `RTX 4070 Ti` `RTX 4070 SUPER` `RTX 4070` `RTX 4060 Ti` `RTX 4060`  |
 |                    | NVIDIA Professional | `L4` `L40` `RTX 6000`                                                                                       |
-| 8.6                | GeForce RTX 30xx    | `RTX 3090 Ti` `RTX 3090` `RTX 3080 Ti` `RTX 3080` `RTX 3070 Ti` `RTX 3070` `RTX 3060 Ti` `RTX 3060`         |
+| 8.6                | GeForce RTX 30xx    | `RTX 3090 Ti` `RTX 3090` `RTX 3080 Ti` `RTX 3080` `RTX 3070 Ti` `RTX 3070` `RTX 3060 Ti` `RTX 3060` `RTX 3050 Ti (notebook)` `RTX 3050 (notebook)`   |
```
Comment (jmorganca) [inline docs/gpu.md:13]:
```suggestion
| 8.6                | GeForce RTX 30xx    | `RTX 3090 Ti` `RTX 3090` `RTX 3080 Ti` `RTX 3080` `RTX 3070 Ti` `RTX 3070` `RTX 3060 Ti` `RTX 3060` `RTX 3050 Ti` `RTX 3050`   |
```

**Comment by bean5** [comment]
Thanks for the suggestion @jmorganca . Back to you now. Great project!

**Comment by bean5** [comment]
I'm not sure why the second commit is not appearing as "Verified". I marked this as allow edit by maintainers, so I could remove that commit and you could apply your suggestion directly. The first commit is acting as verified as expected. Thoughts @jmorganca ?

**Review by dhiltgen — APPROVED** [review]

**Comment by bean5** [comment]
I was just trusting the documentation for my GPU. I was updating my instance today so I can maybe run it with GPU now.

**Pull request merged**